types/src/execution_result.rs

```diff
@@ -287,39 +266,54 @@ impl FromBytes for ExecutionResult {
     }
 }
 
-/// The effect of executing a single deploy.
+/// The journal of execution transforms from a single deploy.
 #[derive(Clone, Eq, PartialEq, Serialize, Deserialize, Default, Debug)]
 #[cfg_attr(feature = "std", derive(JsonSchema))]
 #[serde(deny_unknown_fields)]
-pub struct ExecutionEffect {
+pub struct JsonExecutionJournal {
     /// The resulting operations.
+    #[deprecated(since = "1.4.0")]
     pub operations: Vec<Operation>,
-    /// The resulting transformations.
+    /// The journal of execution transforms.
     pub transforms: Vec<TransformEntry>,
 }
 
-impl ToBytes for ExecutionEffect {
+impl JsonExecutionJournal {
+    /// Constructor for [`JsonExecutionJournal`].
+    #[allow(deprecated)]
+    pub fn new(transforms: Vec<TransformEntry>) -> Self {
+        Self {
+            transforms,
+            operations: Default::default(),
```

Comment:
We were suppossed to discuss it before making the decision - i.e. we can still recover `Op`s from the `Transform`s. If we all agree to make it empty that's fine but let's have that discussion first.

Reply:
cc @EdHastingsCasperLabs @Fraser999 . This change seems unharmful as probably no one uses the `Ops` offchain.